hydra-cluster/test/Test/GeneratorSpec.hs

```diff
@@ -38,21 +34,31 @@ spec = parallel $ do
 
 prop_keepsUTxOConstant :: Property
 prop_keepsUTxOConstant =
-  forAll arbitrary $ \(Positive n) -> do
+  -- FIXME: Breaks with values > 33. Why?
+  forAll (inputs maximumNumberOfParties) $ \(Positive n, clientKeys) -> do
```

Comment:
Must: address this. Nothing should stop us from having more than `maximumNumberOfParties` clients in a benchmark!

The current benchmark just was always the same number of participants as clients, but in "demo mode", we do not care how big the cluster of hydra nodes actually is.